about.html

```diff
@@ -0,0 +1,367 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="main.css">
+  <title>About us</title>
+</head>
+
+<body class="font-Roboto px-20">
+  <header class="navbar">
+    <div class="logo">
+      <img src="./assets/logo.svg" alt="logo" class="logo__img">
+    </div>
+    <ul class="navbar__list">
+      <li class="navbar__item">
+        <a href="./index.html" class="link">Home</a>
+      </li>
+      <li class="navbar__item">
+        <a href="./product.html" class="link">Product</a>
+      </li>
+      <li class="navbar__item">
+        <a href="./pricing.html" class="link">Pricing</a>
+      </li>
+      <li class="navbar__item"><a href="./about.html" class="link">About Us</a></li>
+      <li class="navbar__item"><a href="./blog.html" class="link">Blog</a></li>
+      <li class="navbar__item">
+        <a href="./contact.html" class="link">Contact</a>
+      </li>
+      <li class="navbar__item"><button class="btn">Free trial</button></li>
+    </ul>
+  </header>
+  <main>
+    <header class="header">
+      <div class="header__title">
+        <h1 class="heading heading--xl heading--center">
+          We are proud of our products
+        </h1>
+        <p class="text text--lg text-center text--faded-lg">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+          erat.
+        </p>
+      </div>
+      <div class="blur-gradient absolute bottom-[30%] left-[140px]"></div>
```

Comment:
Instead of putting[30%] and [140px] put them in css file. And remember to use "rem" instead of "px" because you are using tailwind